src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/IndexStatusService.kt

```diff
@@ -34,8 +34,8 @@ class IndexStatusService(
   fun markRebuildStarting(): Boolean {
     val currentIndexStatus = getCurrentIndex()
 
-    if (currentIndexStatus.inProgress) {
-      log.warn("Index marked as already in progress")
+    if (currentIndexStatus.inProgress.and(currentIndexStatus.inError.not())) {
```

Comment:
should this be `inError` rather than `inError.not()`  ?

Reply:
Or maybe message needs changing

Reply:
yes - I think you're right - it's not quite correct. Will update and ensure test picks this up.